systemtest/src/main/java/io/strimzi/systemtest/utils/VerificationUtils.java

```diff
@@ -0,0 +1,463 @@
+                    assertThat(cm.getMetadata().getLabels().get("app"), is(nullValue()));
+                    assertThat(cm.getMetadata().getLabels().get(Labels.STRIMZI_KIND_LABEL), is(KafkaMirrorMaker2.RESOURCE_KIND));
+                } else if (cm.getMetadata().getName().equals(clusterName.concat("-kafka-config"))) {
+                    assertThat(cm.getMetadata().getLabels().get("app"), is(appName));
+                    assertThat(cm.getMetadata().getLabels().get(Labels.STRIMZI_KIND_LABEL), is(Kafka.RESOURCE_KIND));
+                    assertThat(cm.getMetadata().getLabels().get(Labels.STRIMZI_CLUSTER_LABEL), is(clusterName));
+                } else if (cm.getMetadata().getName().equals(additionalClusterName.concat("-kafka-config"))) {
+                    assertThat(cm.getMetadata().getLabels().get("app"), is(appName));
+                    assertThat(cm.getMetadata().getLabels().get(Labels.STRIMZI_KIND_LABEL), is(Kafka.RESOURCE_KIND));
+                    assertThat(cm.getMetadata().getLabels().get(Labels.STRIMZI_CLUSTER_LABEL), is(additionalClusterName));
+                } else {
+                    LOGGER.info("CM {} is not related to current test", cm.getMetadata().getName());
+                }
+            }
+        );
+    }
+
+    /**
+     * Verifies metadata labels for service
+     * @param namespaceName Namespace name where service is located
+     * @param clusterName Name of the cluster linked with service - used for prefix
+     * @param nameLabel "Name" label in the service metadata
+     * @param serviceToVerify Name of the service for verification
+     * @param kind Resource kind of the verified service
+     */
+    public static void verifyServiceLabels(String namespaceName, String clusterName, String nameLabel, String serviceToVerify, String kind) {
+        LOGGER.info("Verifying labels for KafkaConnect Services");
+
+        String serviceName = clusterName.concat("-").concat(serviceToVerify);
+        Service service = kubeClient(namespaceName).getService(serviceName);
+
+        assertThat(service, is(notNullValue()));
+
+        LOGGER.info("Verifying labels for service {}", service.getMetadata().getName());
+        assertThat(service.getMetadata().getLabels().get(Labels.STRIMZI_CLUSTER_LABEL), is(clusterName));
+        assertThat(service.getMetadata().getLabels().get(Labels.STRIMZI_KIND_LABEL), is(kind));
+        assertThat(service.getMetadata().getLabels().get(Labels.STRIMZI_NAME_LABEL), is(clusterName.concat("-").concat(nameLabel)));
+    }
+
+    /**
+     * Verifies metadata labels for service accounts
+     * @param namespaceName Namespace name where service accounts are located
+     * @param clusterName Name of the cluster linked with service accounts
+     * @param appName Name of the app linked with service accounts
+     */
+    public static void verifyServiceAccountsLabels(String namespaceName, String clusterName, String appName) {
```

Comment:
is the `appName` something we should actually check? I saw on multiple places that `appName` is `null`. So why have this check when it always checks a `null` value? (you have it across all of these methods, so I would remove it if it's not needed)

Reply:
Thats good point, if its something we should not thrive to check, its useless here. It can always be changed in future if there is a need for app verification, but for now i'll remove it.